web/sdk/react/views-new/auth/subscribe/subscribe-view.module.css

```
@@ -0,0 +1,56 @@
+  padding: var(--rs-space-8);
+  box-sizing: border-box;
+  box-shadow: var(--rs-shadow-feather);
+  border: 1px solid var(--rs-color-border-base-primary);
+  border-radius: var(--rs-radius-2);
+}
+
+.formField {
+  width: 100%;
+}
+
+.subscribeContainer :global(input::placeholder) {
```

Stylelint (17.9.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)